tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -0,0 +1,144 @@
+from bears.natural_language.WriteGoodLintBear import WriteGoodLintBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """The 70s era was awesome for music lovers.
+""".splitlines(keepends=True)
+
+bad_file = """So the 70s era was awesome for music lovers.
+""".splitlines(keepends=True)
+
+passive_file = """This analysis was provided by coala-bears.
+""".splitlines(keepends=True)
+
+Wordy_file = """It was a project which was very complex in structure and very
```

Comment:
why is the first letter capitalized? shouldn't proper camel case be `wordy_file`?